apps/prairielearn/src/admin_queries/generate_assessment_instances.sql

```diff
@@ -32,23 +32,23 @@ FROM
   assessments AS a
   JOIN course_instances AS ci ON (ci.id = a.course_instance_id)
   JOIN pl_courses AS c ON (c.id = ci.course_id)
-  JOIN group_configs AS gc ON (gc.assessment_id = a.id)
-  JOIN groups AS g ON (g.group_config_id = gc.id)
+  JOIN team_configs AS gc ON (gc.assessment_id = a.id)
+  JOIN teams AS g ON (g.team_config_id = gc.id)
```

Comment:
Nit: are you renaming the aliases too (gc to tc, g to t)? Not sure if it's worth it, just mentioning for completeness.

Reply:
I am not. That would happen in a follow up.